unix/vncconfig/vncExt.h

```diff
@@ -181,7 +181,7 @@ typedef struct {
   CARD8 vncExtReqType; /* always VncExtConnect */
   CARD16 length B16;
   CARD8 strLen;
-  CARD8 pad0;
+  CARD8 viewOnly;
```

Comment:
Hi, I'm not really an expert here, it's also where I ended up when I was looking into this issue myself. I assume this is following a certain specification (in this case some RealVNC extension?). 

From the documentation:
```
Several message formats include padding bits or bytes.  For maximum
compatibility, messages should be generated with padding set to zero,
but message recipients should not assume padding has any particular
value
```
I'm just trying to say that I'm not sure you can change the message format here, even though you are not technically changing it.

Reply:
This is not the RFB protocol; it's the TigerVNC X11 extension. vncconfig interacts with Xvnc an X11 client. The message flow over the X11 connection, usually using the Unix socket at `/tmp/.X11-unix/X<diplay-number>`.

Reply:
I see, this makes much more sense now.